Create data-views.md (#373)
osisoft/OCS-Docs: Content_Portal/add-organize-data/attach-meaning/data-views/data-views.md

---
uid: Dataviews
---

# Data views

Data views are subsets of data from one or more streams. You create a data view by defining the target namespace, and then specifying the source stream or streams, selected data fields, time period, and interpolation interval. 

## Create a data view

To create a new data view, complete the following steps:

> 

Reviewer: Please start all procedures by telling the user how to get to the page from the OCS menu.

1. Click the menu icon and click Data Views (under Analytics).

Author: Done.

   **Note**: Assets are available for use through specific licensing only. If you are interested in using Assets, contact your account team for more details. 
   
6. In the **Search** field of the **Query Value** section, type a portion of the name of any known stream or asset (preceded/followed by the * wildcard symbol). 

   **Note**: If you have created a metadata rule in the same namespace, you can enter a selected metadata key instead of a stream name in the **Search** field to select a more specific set of streams.
   
7. Click **Save**. The query results populate and the **Index Configuration** pane displays.
8. In the **Index Configuration** pane, select the date and time for the **Start Index** and **End Index** to specify the time period of the data to be included in the data view.
9. Enter a value in the **Time Interval** field to specify the time interval by which to display data. The **Time Interval** field provides entry spaces for Days, Hours, Minutes, and Seconds.
10. Select **Save Default with Data View** to save your settings with this data view.

> 11. In the **Field Management** pane, click **+ Add** to select available data fields from the query results to include in the data view. Select the data fields you want to add in the **Add Data Fields** dialog and click **Add**.

Reviewer: ```suggestion
11. In the **Field Management** pane, click **Add** to select available data fields from the query results to include in the data view. Select the data fields you want to add in the **Add Data Fields** dialog and click **Add**.
```
In an earlier step, you tell them to click Add Data View, and you omit the + sign there. I would suggest leaving it off here as well.

---
uid: Dataviews
---

# Data views

Data views are subsets of data from one or more streams. You create a data view by defining the target namespace, and then specifying the source stream or streams, selected data fields, time period, and interpolation interval. 

## Create a data view

To create a new data view, complete the following steps:

1. In the OCS Portal, click the menu icon and select **Data Views** under **Analytics**.
2. In the **Data Views** pane of the OCS Portal, click **Add Data View**.
3. Enter the data view **Name** and **Description** in the **Data View Information** pane.
4. Click **Standard** or **Narrow** to specify the general shape of the data display.
5. In the **Query ID** field in the center pane, enter a name for *Query1* in the **Query ID** field, or accept the default name. 
6. In the **Query Type** section, select **Streams** or **Assets** to indicate which objects you want this query to search.

>    **Note**: Assets are available for use through specific licensing only. If you are interested in using Assets, contact your account team for more details. 

Reviewer: ```suggestion
   **Note**: If assets does not appear as a choice, it means you are not licensed for this feature. If you are interested in using assets, contact your account team for more details. 
```

---
uid: Dataviews
---

# Data views

> Data views are subsets of data from one or more streams. You create a data view by defining the target namespace, the source stream or streams, the desired data fields, time period, and interpolation interval. 

Reviewer 2: I wouldn't use the word "desired". 
```suggestion
Data views are subsets of data from one or more streams. You create a data view by defining the target namespace, then specifying the source stream or streams, the data fields, time period, and interpolation interval. 
```

Author: Done.

## Create a data view

> To create a new data view, complete the following:

Reviewer 2: ```suggestion
To create a new data view, complete the following steps:
```

> 1. In the **Data Views** pane of the OCS Portal, click **Add Data View8*.

Reviewer 2: ```suggestion
1. In the **Data Views** pane of the OCS Portal, click **Add Data View**.
```

Author: Good catch - - fixed.

2. Enter the data view **Name** and **Description** in the **Data View Information** pane.
3. Click **Standard** or **Narrow** to specify the general shape of the data display.
4. In the **Query ID** field in the center pane, enter a name for *Query1* in the **Query ID** field, or accept the default name. 
5. In the **Query Type** section, select **Streams** or **Assets** to indicate which objects you want this query to search.

   **Note**: Assets are available for use through specific licensing only. If you are interested in using Assets, contact your account team for more details. 
   
6. In the **Search** field of the **Query Value** section, type a portion of the name of any known stream or asset (preceded/followed by the * wildcard symbol). 

   **Note**: If you have created a metadata rule in the same namespace, you can enter a selected metadata key instead of a stream name in the **Search** field to select a more specific set of streams.
   
7. Click **Save**. The query results populate and the **Index Configuration** pane displays.

> ## Configure a data view

Reviewer 2: Why is this a separate section of steps? It seems to be a heading in the middle of a procedure. There are no instructions for getting to the Index Configuration pane outside of performing the previous procedure. Would they ever be performed separately?
There is no introduction sentence for the steps.

Author: Yes - - I was thinking of this as separate because you can also select a Data view and click "Edit" and get the Configuration page too, but the fact is that you cannot create one from scratch without having to do the configuration steps. So I'll combine them.

Author: Done.

> 1. In the **Index Configuration** pane, select the date and time for the **Start Index** and **End Index** to specify the time period of the data to be included in the data view.

Reviewer 2: ```suggestion
1. In the **Index Configuration** pane, select the date and time for the **Start Index** and **End Index** to specify the time period of the data to include in the data view.
```

Author: Done.

> 2. Enter a value in the **Time Interval** field to specify the time interval by which to display data. The **Time Interval** field provides entry spaces for Days, Hours, Minutes, and Seconds.

Reviewer 2: This is a different structure than the other steps.
```suggestion
2. In the **Time Interval** field, specify the time interval by which to display data. The **Time Interval** field provides entry spaces for Days, Hours, Minutes, and Seconds.
```

Author: Done.

3. Select **Save Default with Data View** to save your settings with this data view.

> 4. In the **Field Management** pane, click **+ Add** to select available data fields from the query results to include in the data view. Select the data fields you want to add in the **Add Data Fields** dialog and click **Add**.

Reviewer 2: General rule is not to include extra characters in button names **Add** instead of **+ Add**.
```suggestion
4. In the **Field Management** pane, click **Add**. Then in the **Add Data Fields** window, select the data fields from the query results to include in the data view and click **Add**.
```

Author: Done.

> 5. In the **Field Management** pane, use the Up and Down Arrows to move selected data fields up or down in the display, and click **Apply**.

Reviewer 2: I am not sure what the standard for the "up and down arrows" is. Would inline images be possible?

Author: I tried inline images and got them to work partially - - they're inline (no break), but they're huge, and don't respond to my resizing the source image.  Sticking with "Up and Down Arrows" for now, and will bring up how to scale images at Writer's Inquisition.
